### PR TITLE
Enables insert queries

### PR DESCRIPTION
**Pull request by kellyfelkins**
Enables insert queries.

The result of an insert is a tuple: `{:updated, [number of records inserted]}`

### CODE REVIEW

**Comment by zbarnes757** [comment]
@kellyfelkins Thanks to @dustinfarris, the dialyzer issue should be resolved now in v0.3.3

File: lib/snowflex.ex [hunk 1]
```diff
@@ -37,7 +37,7 @@ defmodule Snowflex do
   @type query_opts :: [timeout: timeout(), map_null_to_nil?: boolean()]
 
   @spec sql_query(atom(), String.t(), query_opts()) ::
-          sql_data() | {:error, term()}
+          sql_data() | {:error, term()} | {:updated, term()}
```
Comment (dustinfarris) [inline lib/snowflex.ex:40]: These should be `{:updated, integer()}` to match what odbc returns.

**Review by dustinfarris — CHANGES_REQUESTED** [review]
please add unit tests for the updated return.  you can add a pattern match to the mock worker in [connection_test.exs](https://github.com/pepsico-ecommerce/snowflex/blob/55ad030576d6af69f2fd44b49f68695b74f7b2a2/test/snowflex/connection_test.exs#L18), e.g something like.

```exs
def handle_call({:sql_query, "dummy insert query"}, _from, state) do
  {:reply, {:ok, {:updated, 123}}, state}
end
```

**Review by zbarnes757 — COMMENTED** [review]
LGTM but can you add your changes to `CHANGELOG.md` under the unreleased section? When I create a new release, I'll tag it with the release number.

**Review by dustinfarris — APPROVED** [review]
LGTM.  Thanks!

**Pull request merged**